src/v2/ion/ui-schema/ion-boolean-handler.js

```diff
@@ -32,12 +32,13 @@ const getRadioUiSchema = ({ label, required, options }) => ({
   sublabel: required? null : loc('oie.form.field.optional', 'login'),
 });
 
-const getCheckboxUiSchemaWithDefaultValue = ({ label, type, required }) => ({
+const getCheckboxUiSchemaWithDefaultValue = ({ label, type }) => ({
```

Comment:
test? already covered? alternatively you could keep the param and set default value to `true`

Reply:
yeah there is a test covered this new uiSchema. I just did an end to end test to confirm that it works as expected. shown as the screen record in box link. Thanks!